### PR TITLE
Add new field sessionId into Session

### PR DESCRIPTION
**Pull request by vuhainguyen**
CDF-13661

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -8,7 +8,7 @@ val circeVersion = "0.13.0"
 val sttpVersion = "3.3.15"
 val Specs2Version = "4.6.0"
 val artifactory = "https://cognite.jfrog.io/cognite/"
-val cogniteSdkVersion = "1.5.18"
+val cogniteSdkVersion = "1.5.19-SNAPSHOT"
```
Comment (vuhainguyen) [inline build.sbt:11]: TODO Update this once https://github.com/cognitedata/cognite-sdk-scala/pull/413 is merged

File: build.sbt [hunk 2]
```diff
@@ -20,7 +20,7 @@ lazy val commonSettings = Seq(
   organization := "com.cognite.spark.datasource",
   organizationName := "Cognite",
   organizationHomepage := Some(url("https://cognite.com")),
-  version := "1.4.46",
+  version := "1.4.47-SNAPSHOT",
```
Comment (vuhainguyen) [inline build.sbt:23]: TODO Remove SNAPSHOT

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,9 @@
+# 1.4.47
+
+## Fixes
+
+* Add sessionId into Session as it is requires when refreshing the session
```
Comment (hakontro) [inline CHANGELOG.md:5]:
```suggestion
* Add the new field `sessionId` to session as it is required when refreshing the session.
```

**Review by hakontro — APPROVED** [review]
LGTM, feel free to merge once SDK is released.

**Pull request merged**